package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "corkjs",
+  "version": "1.0.0",
+  "description": "Displays information about CorkJs powered by meetup.com api",
+  "main": ".eslintrc.js",
```

Comment:
Should probably remove the `main` field altogether. I don't think it's required anyway.